spec/factories/users.rb

```diff
@@ -12,5 +12,9 @@
       email { "user#{rand(1...9999)}@mail.com" }
       role { "publisher" }
     end
+    factory :subscriber do
+      email { "user#{rand(1...9999)}@mail.com" }
```

Comment:
If it's the same way to create an email, then it should be enough if you have it in the main factory, right?

Reply:
Fixed.